README.md

```diff
@@ -212,16 +212,23 @@ The console output is designed to make it easy to copy and paste into a spreadsh
 
 ### Jupyter Notebook
 
+#### Prerequisites
 If you want to generate graphs in a Jupyter Notebook, there is some additional setup that you need to follow.
 
-1. Install Jupyter-Lab or nteract
-1. Install dontet/interactive
+1. Install [Jupyter-Lab](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) or nteract.
+1. Install [dontet/interactive.](https://github.com/dotnet/interactive/blob/main/docs/NotebooksLocalExperience.md)
 
+#### Sample Notebook
 A [sample notebook](https://github.com/corgibytes/freshli/blob/main/Sample.ipynb) can be found in this repository.
+To run the statements within this sample file, you will need to pull down a local version of
+[XPlot]() and run the build script located inside of this project. Note that, currently,
```

Comment:
I think you are missing the link to XPlot.  Do we need specific version?

Reply:
The version of Xplot used in the Sample is the version that is in the Corgibytes Repo

Reply:
I can add a link

Reply:
If we remove the customer XPlot requirement than I think we can remove the documentation about install XPlot.